src/main/java/io/confluent/idesidecar/restapi/models/graph/FlinkComputePool.java

```diff
@@ -0,0 +1,17 @@
+package io.confluent.idesidecar.restapi.models.graph;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.quarkus.runtime.annotations.RegisterForReflection;
+
+@RegisterForReflection
+public record FlinkComputePool(
+    @JsonProperty("id") String id,
+    @JsonProperty("spec") FlinkComputePoolSpec spec,
+    @JsonProperty("status") FlinkComputePoolStatus status,
+    @JsonProperty("connectionId") String connectionId
+) {
```

Comment:
What about modeling this record similar to the other records in this package, like `CCloudKafkaCluster`, so that it exposes the information from the `spec` and `status` at the top level? Maybe something like:

```java
public record FlinkComputePool(
    String id,
    String name,
    String cloud,
    String region,
    Integer maxCfu,
    CCloudReference organization,
    CCloudReference environment,
    String connectionId
) { }
```

You could then move `FlinkComputePoolSpec` and `FlinkComputePoolStatus` to the `RealCCloudFetcher`.

Reply:
done